databaseconnect.py

```diff
@@ -128,21 +130,21 @@ def get_question_response(subject, root, verb):
         res = cur.fetchall()
         found = 0
         for r in res:
-            if r[-1] == str(subject):
+            if r[-1] == str(subject[0]):
                 found = 1
                 break
         if found == 1:
-            cur.execute(f"SELECT verb FROM statement_table WHERE subject='{subject}'")  # noqa: E501
+            cur.execute("SELECT verb FROM statement_table WHERE subject= '{}'".format(subject[0]))  # noqa: E501
             res = cur.fetchone()
-            checkVerb = res[0]   # checkVerb is a string while verb is a list. checkVerb ['verb']   # noqa: E501
+            checkVerb = res[0]   # checkVerb is a string while verb is a list. checkVerb ['verb']  # noqa: E501
             if checkVerb == '[]':
-                cur.execute(f"SELECT sentence FROM statement_table WHERE subject='{subject}'")  # noqa: E501
+                cur.execute("SELECT sentence FROM statement_table WHERE subject= '{}'".format(subject[0]))  # noqa: E501
                 res = cur.fetchone()
                 B = res[0]
                 return B, 0
             else:
                 if checkVerb[2:-2] == verb[0]:
-                    cur.execute(f"SELECT sentence FROM statement_table WHERE subject='{subject}'")  # noqa: E501
+                    cur.execute("SELECT sentence FROM statement_table WHERE subject= '{}'".format(subject[0])) # noqa: E501
```

Comment:
Strings like these need to go. :slightly_smiling_face: 

Have to try the following type of `cur.execute` statements instead.
```python
insert_stmt = (
  "INSERT INTO employees (emp_no, first_name, last_name, hire_date) "
  "VALUES (%s, %s, %s, %s)"
)
data = (2, 'Jane', 'Doe', datetime.date(2012, 3, 23))
cursor.execute(insert_stmt, data)

select_stmt = "SELECT * FROM employees WHERE emp_no = %(emp_no)s"
cursor.execute(select_stmt, { 'emp_no': 2 })
```
Sources: [MySQL docs](https://dev.mysql.com/doc/connector-python/en/connector-python-api-mysqlcursor-execute.html)